src/pvecontrol/config.py

```diff
@@ -10,16 +11,30 @@
             "host": str,
             "user": str,
             "password": str,
+            "timeout": int,
             "node": confuse.Optional(
                 {
                     "cpufactor": confuse.Optional(float, None),
                     "memoryminimum": confuse.Optional(int, None),
                 },
                 default={},
             ),
+            "sanitycheck": {
+                VmBackups.id: confuse.Optional(
+                    {
+                        "max_last_backup_in_minutes": int,
+                    },
+                    default={},
+                ),
+            },
         }
     ),
     "node": {"cpufactor": float, "memoryminimum": int},
+    "sanitycheck": {
```

Comment:
Instead of sanitycheck like the above option maybe name it `vm`.`max_last_backup`.  no need to add in_minutes in desc. this is only in documentation.